examples/private_service_connect_google_apis/main.tf

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+terraform {
+  required_providers {
+    google = {
+      version = ">= 3.45.0"
+    }
+    null = {
```

Comment:
We don't seem to be using this provider in config?

Reply:
I confirmed that this example doesn't work without google-beta, so I kept that and removed the other provider block.